code/game/turfs/open/floor/mineral_floor.dm

```diff
@@ -174,9 +174,6 @@
 /turf/open/floor/mineral/plastitanium/red/snow_cabin
 	temperature = 180
 
-/turf/open/floor/mineral/plastitanium/red/brig
```

Comment:
make an updatepaths for this change https://github.com/tgstation/tgstation/tree/master/tools/UpdatePaths